### PR TITLE
docs: switch to gopher favicon

### PR DESCRIPTION
**Pull request by rfjakob**
What is the purpose of this change? What does it change?
--------------------------------------------------------

The favicon on restic.readthedocs.io still contained the old
superman-style logo.

This changes it to the 32x32 icon also used on https://forum.restic.net/ ,
just converted to .ico using Gimp.

Was the change discussed in an issue or in the forum before?
------------------------------------------------------------

Yes, https://forum.restic.net/t/new-logo-for-restic/394

Checklist
---------

- [x] I have read the [Contribution Guidelines](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#providing-patches)
- [ ] I have added tests for all changes in this PR
- [ ] I have added documentation for the changes (in the manual)
- [ ] There's a new file in `changelog/unreleased/` that describes the changes for our users (template [here](https://github.com/restic/restic/blob/master/changelog/TEMPLATE))
- [ ] I have run `gofmt` on the code in all commits
- [x] All commit messages are formatted in the same style as [the other commits in the repo](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#git-commits)
- [x] I'm done, this Pull Request is ready for review

### CODE REVIEW

**Comment by moritzdietz** [comment]
Thanks for this PR! Quick question: Did you convert the `.ico` file yourself or did you copy the one used for the forum?
I am not sure if this is actually worth of mentioning in a changelog entry as it does not affect restic itself. It doesn't hurt but I am also not sure if it's actually needed/wanted by the maintainers.

**Comment by rfjakob** [comment]
The forum uses a png, i took the png and converted to ico with gimp.

File: changelog/unreleased/pull-2337 [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Documentation: switch to gopher favicon
+    
+The favicon on restic.readthedocs.io still contained the old
```
Comment (moritzdietz) [inline changelog/unreleased/pull-2337]:
```suggestion
The favicon used on restic.readthedocs.io was still the old
```

File: changelog/unreleased/pull-2337 [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+superman-style logo.
+
+We changed it to the 32x32 gopher icon also used on https://forum.restic.net/ ,
```
Comment (moritzdietz) [inline changelog/unreleased/pull-2337]:
```suggestion
We changed it to the 32x32 gopher icon also used on https://forum.restic.net/
```

File: changelog/unreleased/pull-2337 [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+just converted to .ico using Gimp.
```
Comment (moritzdietz) [inline changelog/unreleased/pull-2337]:
```suggestion
```

**Review by moritzdietz — COMMENTED** [review]
So if the maintainers decide to keep this CHANGELOG entry, I would word it differently.
See my suggested change. I don't think it is relevant how you modified the `.png` to get the favicon.

**Review by rawtaz — CHANGES_REQUESTED** [review]
Wow, there certainly seems to be *a lot* to take into account with favicon, if you read https://stackoverflow.com/a/23734416. For constructive reasons I think we can just replace the file for now, if there's trouble with it down the road, let's address it then.

@rfjakob I agree with @moritzdietz that for this type of change, an entry in the changelog adds little value. Can you please remove those changes (keeping only the new favicon file), and amend/squash so there's just one commit? Let us know if you need help 👍 

Thanks!

**Review by rawtaz — APPROVED** [review]
I removed the changelog file (not needed for documentation changes), thank you very much for this nice little piece of contribution! :)

**Pull request merged**